UI/Web/src/app/manga-reader/_components/infinite-scroller/infinite-scroller.component.ts

```diff
@@ -300,6 +305,19 @@ export class InfiniteScrollerComponent implements OnInit, OnChanges, OnDestroy,
       || 0);
   }
 
+  onWheel(event: WheelEvent) {
+    if (!this.heightLessThanView()) {
+      // Only execute on small webtoons.
+      return;
+    }
+
+    if (event.deltaY > 0) {
+      this.loadNextChapter.emit();
```

Comment:
I didn't update the value of `this.scrollingDirection` here -- not sure if this is needed. :)

Reply:
Yeah, I think it is a good idea so you know the direction you were last scrolling. It wont matter much, but better to have.